pkg/commands/build/status.go

```diff
@@ -125,7 +125,7 @@ func displayBuildStatus(cmd *cobra.Command, bld v1alpha2.Build) error {
 	}
 
 	statusItems := []string{
-		"Image", bld.Status.LatestImage,
+		"Image Resource", bld.Status.LatestImage,
```

Comment:
This is explicitly not the image resource it is the image itself.

Reply:
> Could we avoid the phrasing resource configuration?

yes!